rocks-bin/src/list.rs

```diff
@@ -10,13 +10,10 @@ pub struct ListCmd {
     porcelain: bool,
 }
 
-pub fn list_installed(list_data: ListCmd, config: &Config) -> Result<()> {
+pub fn list_installed(list_data: ListCmd, config: Config) -> Result<()> {
```

Comment:
note: I'll try to remove references to the config in all other invocations in `main.rs` for consistency.